src/main/kotlin/frc/robot/subsystems/wrist/WristConstants.kt

```diff
@@ -0,0 +1,10 @@
+package frc.robot.subsystems.wrist
+
+import edu.wpi.first.units.measure.Angle
+import frc.robot.lib.extensions.degrees
+
+enum class Angles(degrees: Angle) {
+    resetAngle(0.0.degrees),
+    openAngle(0.0.degrees),
+    closeAngle(0.0.degrees)
```

Comment:
```suggestion
    RESET(0.deg),
    CLOSE(0.deg),
    OPEN(0.deg)
```
Reset is a bit unclear, though. What exactly does it mean?